src/hooks/useCountdownSeconds.ts

```diff
@@ -1,27 +1,44 @@
 import { useState } from "react"
 
-const useCountdownSeconds = (seconds: number) => {
-  const [sec, setSec] = useState(seconds)
+const useCountdownSeconds = (seconds?: number, onCountdown?: () => void) => {
```

Comment:
This hook starts to get more and more complex hehe, what do you think, should we just use the [`useCountdown`](https://usehooks-ts.com/react-hook/use-countdown) hook from `usehooks-ts`? I think we'll add this package to our app anyways, since we'll need some hooks of it when we migrate from Chakra. That way we wouldn't really need to maintain these kind of hooks, just use the premade ones from this package.

Reply:
Nice, added the lib & changed the hook usage in https://github.com/guildxyz/guild.xyz/pull/1301/commits/0321fb71d7c04e39b672601fd2599b9c424b1c7c
I didn't remove the hook yet, but I'll do that in a follow-up PR :ayy:

One thing, which I'm not sure is that the deadline is a little less accurate since we need to pass it in the hook call, but it shouldn't be a problem, just something to keep in mind